src/app/globals.css

```
@@ -1305,6 +1305,42 @@
+    }
+  }
+
+  .streaming-caret > *:last-child::after,
+  .streaming-caret:empty::after {
+    content: "";
+    display: inline-block;
+    width: 2px;
+    height: 1em;
+    margin-left: 2px;
+    border-radius: 1px;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)